build.gradle

```diff
@@ -10,6 +9,12 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.jfrog.bintray" version "1.7.3"
+}
+
+apply plugin: 'com.github.ben-manes.versions'
```

Comment:
https://github.com/ben-manes/gradle-versions-plugin -> shows what versions you're using vs. what the latest is